app/components/Navbar.tsx

```diff
@@ -22,6 +22,11 @@ const Navbar = () => {
   const currentTheme = theme === "system" ? systemTheme : theme;
   const [showMobileNav, setShowMobileNav] = useState<boolean>(false);
   const pathname = usePathname();
+  const [icons, setIcons] = useState({
+    menuIcon: menuButton,
+    downloadIcon: download,
```

Comment:
Take the time to descriptively name your variables. When I was reviewing your code, I had to come back and reference it here to know it was a download Icon. Why not just name it "downloadIcon"?